src/Orleans.CodeGenerator/InvokableGenerator.cs

```diff
@@ -92,8 +92,8 @@ public GeneratedInvokableDescription Generate(InvokableMethodDescription invokab
 
             static Accessibility GetAccessibility(IMethodSymbol methodSymbol)
             {
+                Accessibility accessibility = methodSymbol.DeclaredAccessibility;
```

Comment:
If the type is internal and the method is public, what will the value of `DeclaredAccessibilty` be?

Reply:
That's gonna be public, which will be downgraded to internal after the loop below.